stripe/stripe_object.py

```diff
@@ -237,7 +238,9 @@ def serialize(self, previous):
             elif isinstance(v, stripe.api_resources.abstract.APIResource):
                 continue
             elif hasattr(v, 'serialize'):
-                params[k] = v.serialize(previous.get(k, None))
+                child = v.serialize(previous.get(k, None))
+                if child:
```

Comment:
I'm not sure if it's possible in practice, but in theory `serialize` could return an empty string `""` to unset the entire attribute, and because empty strings are falsey in Python this would get ignored. It might be better to explicitly compare the returned value to an empty dict.

Reply:
Makes sense.
Should have a test for it then, too.

Reply:
> if it's possible in practice

The two internal `serialize` methods will return a dict always, so it is unlikely to come from there.

Added a test with a specialized StripeObject, which is likely what you had in mind?!

Reply:
Yes, exactly. Thanks again!